src/dev/flang/tools/docs/Html.java

```diff
@@ -218,6 +220,68 @@ private static String source(AbstractFeature feature)
       .replace("$1", featureURL(feature));
   }
 
+  static String processComment(String name, String s)
+  {
+    var codeNo = new ArrayList<Integer>();
+    var codeLines = new ArrayList<String>();
+    var resultLines = new ArrayList<String>();
+
+    s.lines().forEach(l ->
+      {
+        if (l.startsWith("    "))
+          {
+            /* code comment */
+            codeLines.add(l);
+          }
+        else if (l == "")
+          {
+            /* avoid adding lots of line breaks after code comments */
+            if (codeLines.isEmpty()) {
+              resultLines.add(l);
+            }
+          }
+        else
+          {
+            if (!codeLines.isEmpty())
+              {
+                /* dump codeLines into a flang.dev runcode box */
+                var id = "fzdocs." + name + codeNo.size();
+                var code = codeLines
+                  .stream()
+                  .map(cl -> { return cl.replaceAll("^    ", ""); })
+                  .collect(Collectors.joining(System.lineSeparator()));
+                var runCodeWrapper = """
```

Comment:
This string should be moved to the beginning of this java file or even to a dedicated file with html templates and assigned to a `static final String SOME_GOOD_NAME = """...`.